sbt-bridge/test/xsbt/TestDriver.scala

```diff
@@ -0,0 +1,15 @@
+package xsbt
+
+import dotty.tools.dotc._
+import core.Contexts._
+
+class TestDriver extends Driver {
```

Comment:
Sure, but it's a test so it doesn't matter much, and I'd like to avoid changing `ScalaCompilerForUnitTesting` too much since it's adapted from sbt and I synchronize it with the sbt version sometimes.